src/xrpld/app/tx/detail/OfferStream.cpp

```diff
@@ -272,6 +272,19 @@ TOfferStreamBase<TIn, TOut>::step()
             continue;
         }
 
+        bool const deepFrozen = isDeepFrozen(
```

Comment:
should we gate this with the amendment? - altho not necessary since deep freeze flags can never be set if the amendment is not enabled. just a thought

Reply:
yes, the will be no deep frozen trustline before the amendment so there's no problem with this I believe